cip-TOKEN-STD/cip-meiersi-Canton-Network-Token-Standard.md

```diff
@@ -0,0 +1,574 @@
+<pre>
+  CIP:  CIP XXXX
+  Layer: Daml
+  Title: Canton Network Token Standard
+  Author: Simon Meier, Matteo Limberto, Moritz Kiefer
+  License: CC-1.0
+  Status: Draft
+  Type: Standards Track
+  Created: 2025-03-07
+  Post-History: https://lists.sync.global/g/cip-discuss/topic/110627661#msg13
+</pre>
+
+
+## Abstract
+
+This CIP proposes standard APIs for Canton Network tokens so that wallets and
+apps can use them and build on them in a uniform way, i.e., working only against the standard APIs without depending on the specifics of an individual asset.
+The APIs enable four key functionalities:
+
+1. wallets can serve a portfolio view with transaction history
+2. investors can use wallets to initiate direct peer-to-peer transfers and delivery-vs-payment (DVP) transfers
```

Comment:
Probably minor, but it sounds like the wallet is used to initiate a DvP, whereas initiation of the DvP is usually orchestrated in e.g. a trading app.

Maybe something like

 ```suggestion
2. investors can use wallets to initiate direct peer-to-peer transfers and fulfil delivery-vs-payment (DVP) transfers
```